src/DateTime/tests/test_datetime.py

```diff
@@ -683,6 +683,13 @@ def test_security(self):
         self.assertEqual(dt.__roles__, None)
         self.assertEqual(dt.__allow_access_to_unprotected_subobjects__, 1)
 
+    @unittest.skipUnless(PY3K, 'format method is Python 3 only')
+    def test_format(self):
+        dt = DateTime()
+        fmt = '%-d.%-m.%Y %H:%M'
+        result = dt.strftime(fmt)
+        self.assertEqual(result, '{:%-d.%-m.%Y %H:%M}'.format(dt))
```

Comment:
Comparing with an actual result string makes it easier to read the test and to see that it actually does what it is expected.

Additionally `__format__` has some code branches which are not covered here (`fmt` is no string or an empty string.)  Please write additional tests for them.

Reply:
> Comparing with an actual result string makes it easier to read the test and to see that it actually does what it is expected.

That's what I did initially but changed it according to a suggestion from dataflake. The problem with the actual f-string is that it produces errors in the tests for earlier Python versions.

I initially also had a check for the Python version, but this created problems with coverage. As far as I understand, I can't fine-tune the pragma comment for coverage, but I can at least allow the test only for Python 3.